src/test/java/org/opensearch/knn/index/engine/lucene/LuceneMethodResolverTests.java

```diff
@@ -178,21 +178,6 @@ public void testResolveMethod_whenInvalid_thenThrow() {
             )
         );
 
-        // Invalid compression
-        expectThrows(
```

Comment:
Let's keep this in and change to x16 to still throw with a comment explaining that x32 added in this PR